manifests/e/EasyABC/EasyABC/1.3.8.6/EasyABC.EasyABC.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: EasyABC.EasyABC
+PackageVersion: 1.3.8.6
+PackageLocale: en-US
+Publisher: Seymour Shlien
+PackageName: EasyABC
```

```suggestion
PackageName: EasyABC
PackageUrl: https://sourceforge.net/projects/easyabc/
```
It's the first link from an Internet search.